packages/experimental/Button/src/Button.styling.ts

```diff
@@ -2,44 +2,50 @@ import { buttonName, ButtonTokens, ButtonSlotProps, ButtonProps } from './Button
 import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
 import { borderStyles, layoutStyles, fontStyles } from '@fluentui-react-native/tokens';
 import { buttonStates, defaultButtonTokens } from './ButtonTokens';
+import { TextProps } from '@fluentui-react-native/experimental-text';
+import { IViewWin32Props } from '@office-iss/react-native-win32';
```

Comment:
Using IViewWin32Props is driven by the root style for this button being defined as using IViewWin32Props in Button.types.ts: https://github.com/microsoft/fluentui-react-native/blob/3c0f5496f0daeb748c6b2daac8d0465a1d58a672/packages/experimental/Button/src/Button.types.ts#L118

I think using IViewProps makes sense, but I think it's out of scope for this change since it's a change to Button and not for supporting ColorValue..